selfdrive/camerad/snapshot/snapshot.py

```diff
@@ -33,7 +33,7 @@ def snapshot():
   ps = subprocess.Popen("ps | grep camerad", shell=True, stdout=subprocess.PIPE)
   ret = list(filter(lambda x: 'grep ' not in x, ps.communicate()[0].decode('utf-8').strip().split("\n")))
   if len(ret) > 0:
-    params.put("IsTakingSnapshot", "0")
+    put_nonblocking("IsTakingSnapshot", "0")
```

Comment:
I don't think we need the asynchronous version here since it's not in a timing critical loop.